docs/HOW_IT_WORKS.md

```diff
@@ -0,0 +1,34 @@
+# How ngx-prefetch works
+
+After adding a service worker to your project, as specified in the [prerequisites](README.md#prerequisites), the modifications include 
+the creation of a service worker configuration file called `ngsw-config.json`, which specifies the caching behaviors. 
+Then the build process creates the manifest file `ngsw.json` using information from `ngsw-configuration.json`.

```

Comment:
ngsw-configuration.json -> ngsw-config.json

we should mention that ngsw.json is in the dist folder